templates/taxi/manufacturer_list.html

```diff
@@ -8,6 +9,11 @@ <h1>
     </a>
   </h1>
 
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" value="Search" class="btn btn-secondary">
+  </form>
```

Comment:
What about adding this part into separate file like includes/search.html and including this template in all other needed templates?